docs/additional-functionality/cache-serializer.md

```diff
@@ -29,12 +29,40 @@ nav_order: 2
   `spark.sql.inMemoryColumnarStorage.enableVectorizedReader` will not be honored as the GPU
   data is always read in as columnar. If `spark.rapids.sql.enabled` is set to false
   the cached objects will still be compressed on the CPU as a part of the caching process.
-  Also note that Parquet doesn't support CalendarIntervalType or NullType out of the box, but
-  ParquetCachedBatchSerializer does by decomposing intervals to struct containing the
-  months, days and microseconds and NullType to Int column containing nulls.
-
+  Please note that Parquet doesn't support columns of type CalendarIntervalType or NullType out of
+  the box, but ParquetCachedBatchSerializer does by decomposing intervals to struct containing the
+  months, days and microseconds and NullType to Int column containing nulls. Also note that
```

Comment:
This seems like too much implementation detail because the user won't be able to access the underlying representation.  Personally I'd not mention it at all or at best say CalendarIntervalType and NullType are supported and leave it at that.  But if the detail stays:
```suggestion
  the box, but ParquetCachedBatchSerializer does by decomposing an interval to a struct containing the
  months, days and microseconds and NullType to an Int which is always null. Also note that
```